OIPA/api/organisation/tests/test_organisation_serializers.py

```diff
@@ -46,6 +46,20 @@ def test_OrganisationSerializer(self):
         for field in required_fields:
             assert field in serializer.data, msg.format(field)
 
+    def test_OrganisationDifficultNameSerializer(self):
```

Comment:
Please don't ever mix underscores and camel case.
Function names should be one or another. https://www.python.org/dev/peps/pep-0008/#function-names
I'll accept it for now, since this style is used in all tests :/